src/outputs/output_utils.cpp

```diff
@@ -119,21 +119,23 @@ std::vector<Real> ComputeXminBlocks(Mesh *pm) {
 std::vector<int64_t> ComputeLocs(Mesh *pm) {
   return FlattenBlockInfo<int64_t>(
       pm, 3, [=](MeshBlock *pmb, std::vector<int64_t> &locs, int &i) {
-        locs[i++] = pmb->loc.lx1();
-        locs[i++] = pmb->loc.lx2();
-        locs[i++] = pmb->loc.lx3();
+        auto loc = pmb->pmy_mesh->Forest().GetAthenaCompositeLocation(pmb->loc);
+        locs[i++] = loc.lx1();
+        locs[i++] = loc.lx2();
+        locs[i++] = loc.lx3();
```

Comment:
Yes,  I think in the future it will just need to be 
```c++
auto &loc = pmb->loc;
locs[i++] = loc.tree();
locs[i++] = loc.lx1();
locs[i++] = loc.lx1();
locs[i++] = loc.lx1();
```
or something.

Reply:
Cool. Maybe we add a TODO comment and call it good for now? (Just so I keep it in mind while plumbing?